ADA/ada.qmd

```diff
@@ -21,55 +21,124 @@ It brings together:
 *	a collection of data reports and dashboards
 *	a news and updates area.
 
-As long as you're connected to the network - it's available to anyone who works for the Department of Education.
+As long as you're connected to the network - it's available to anyone who works for the Department of Education. 
 
-See these [Databricks support notebooks](https://adb-2220072380334347.7.azuredatabricks.net/browse/folders/4213691640199507?o=2220072380334347) for more detailed reference guides into how we have setup the service here at DfE, including more information on workspaces, environments, notebooks, clusters, catalogs etc. Please note that you will need a Databricks account to access these. 
+Databricks is currently in its onboarding phase, which is expected to continue until **???**. At the moment, teams can onboard voluntarily until **???**. Following this date, the remaining teams will have their data migrated for them. 
 
+See these [Databricks support notebooks](https://adb-2220072380334347.7.azuredatabricks.net/browse/folders/4213691640199507?o=2220072380334347) for more detailed reference guides into how the service has been set up at DfE, including more information on workspaces, environments, notebooks, clusters, catalogs etc. Please note that you will need a Databricks account to access these. 
 
----
+From the [ADA homepage](https://analytical-data-access.education.gov.uk/) you will be able to find data and access cloud analytical tools. This includes:
 
-# Plans for the future
+* Navigation to Data Discovery Platform to find information regarding metadata of the datasets, including how to request access and information about the quality of the data. The catalogue is supported by the Metadata Team and will eventually include all Department data, as well as tagging of datasets available through ADA.
 
----
+* Analytical workbench to analyse and model data. This is underpinned by Databricks, a cloud-based platform to load and store data; and perform analysis, modelling and machine learning of data; as well as output to various formats to share analysis. The ADA team are looking to extend integration to R Studio in the coming months.
 
-The ambition is to enable access to all datasets across DfE and ESFA and retire the legacy applications. This will create common ways of working.
+* A repository of Reports and Dashboards where outputs of data and analytical work will be saved. This will allow colleagues across DfE to interrogate and visualise data. In time this catalogue will grow and the ADA team are currently building out a formal strategy for this area.
 
 ---
 
-# Useful links
+## What does the ADA project mean for analysts?
 
 ---
```

Comment:
need to undo line break (see earlier comment)

![image](https://github.com/user-attachments/assets/4e8c42b8-1347-4443-86cc-4dda383885d3)